pkg/systemd/services/services.jsx

```diff
@@ -189,7 +189,7 @@ class ServicesPageBody extends React.Component {
             failed: _("Failed to start"),
             activating: _("Running"),
             deactivating: _("Not running"),
-            maintainance: _("Maintanance"),
+            maintenance: _("Maintenance"),
```

Comment:
This is a change in code. It should be reviewed. Usually when codespell finds these changes, it changes the code where it is referenced. But I don't see it elsewhere in this file, so it might be used elsewhere. Or perhaps it was unused or is a bug elsewhere?

Reply:
It's a very rare corner case of a systemd unit state, we don't use/test it anywhere. That table is just for completeness. So this is correct.